src/winforms/toga_winforms/widgets/multilinetextinput.py

```diff
@@ -27,3 +28,7 @@ def get_value(self):
     def rehint(self):
         self.interface.intrinsic.width = at_least(self.interface.MIN_WIDTH)
         self.interface.intrinsic.height = at_least(self.interface.MIN_HEIGHT)
+
+    def winforms_text_changed(self, sender, event):
+        if self.interface._on_change:
+            self.interface._on_change(self.interface)
```

Comment:
You should call here to the `on_change` property instead of the `_on_change` private attribute.

Reply:
ok, done